accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -895,6 +900,7 @@ public static Cleanup shouldCleanup(TxnId txnId, Status status, Durability durab
             return Cleanup.ERASE;
         }
 
+        // TODO (review): If we always send the full route when will this happen?
```

Comment:
`Route.isFullRoute` allows `null`... but at this point it "shouldn't" be null?  Im in favor of keeping it as is